packages/web/partials/header.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="Container">
 
     <h1 class="typography-heading-xlarge-text">Spirit Design System</h1>
-    <h2 class="typography-heading-medium-text text-secondary-inverted-disabled mb-900">Web — Development Preview</h2>
+    <h2 class="typography-heading-medium-text text-primary-inverted-disabled mb-900">Web — Development Preview</h2>
 
     <a href="https://github.com/lmc-eu/spirit-design-system" class="Button Button--inverted Button--large mr-400">
```

Comment:
Ah! This PR reminds me:
![image](https://user-images.githubusercontent.com/33354913/228759775-f9dc6f4c-da27-4c82-ad71-114b753c2e8f.png) – apply bottom offset to the button elements?